back/src/forms/resolvers/forms/intermediary.ts

```diff
@@ -0,0 +1,18 @@
+import { FormResolvers } from "../../../generated/graphql/types";
+import prisma from "../../../prisma";
+
+const intermediaryCompaniesResolver: FormResolvers["intermediaries"] =
+  async form => {
+    const found = await prisma.form.findUnique({
+      where: { id: form.id },
+      select: {
+        intermediaries: true
+      }
+    });
```

Comment:
Encore une fois pour une histoire de dataloaders, je pense qu'il vaut mieux faire:
```suggestion
    const intermediaries = await prisma.form.findUnique({
      where: { id: form.id },
    }).intermediaries();
```